client/my-sites/plan-features-2023-grid/hooks/npm-ready/data-store/use-grid-plans.tsx

```diff
@@ -27,9 +27,11 @@ import type { PricedAPIPlan } from '@automattic/data-stores';
 import type { TranslateResult } from 'i18n-calypso';
 
 // TODO clk: move to plans data store
+// Note: This type is duplicated in ../../../types.ts. Should prob be defined just once.
 export type TransformedFeatureObject = FeatureObject & {
 	availableForCurrentPlan: boolean;
 	availableOnlyForAnnualPlans: boolean;
+	isHighlighted?: boolean;
 };
```

Comment:
I think this type should be deleted here, and imported from types.ts.

Reply:
I will handle this in a follow up.

Reply:
This is fine. This should be preferred (I am removing the other one in another PR). The types in this file will migrate to different locations down the line.